packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -482,8 +485,12 @@ describe("calcite-list", () => {
       // Workaround for page.spyOnEvent() failing due to drag event payload being serialized and there being circular JSON structures from the payload elements. See: https://github.com/Esri/calcite-design-system/issues/7643
       await page.$eval("calcite-list", (list: HTMLCalciteListElement) => {
         (window as TestWindow).calledTimes = 0;
```

Comment:
Suggestion: can you store the typed window in a local `win` variable or similar to reduce typecasting lines?